.vale/styles/style_guide/OxfordComma.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Suggestion: style_guide.OxfordComma.yml
+
+# Use the Oxford comma, this may not work as intended for complex sentences with many commas
```

Comment:
What happens if it doesn't work as intended? Do we get an error anyway? Do we have any examples in our current content?